package.json

```diff
@@ -62,7 +62,7 @@
     "@celo/flake-tracker": "0.0.1-dev"
   },
   "engines": {
-    "node": "12"
+    "node": ">=18"
```

Comment:
I wonder if the >= part is over-promising. not sure what breaking changes will be in node 20 and 22 that we'll have to build in compatibility for in our code and in all the dependencies